syntax = "proto3";

message Empty {}

message ShowtimeData {
    string date = 1;
    repeated ShowtimeMovieData movies = 2;
}

message ShowtimeMovieData {
    string id = 1;
    int32 seatsMax = 2;
    int32 seatsBooked = 3;
}
